src-tauri/src/mocks/mocks.rs

```diff
@@ -120,20 +112,34 @@ pub fn generate_loc_info(num_nodes: i32) -> HashMap<u32, MeshNode> {
             environment_metrics: vec![],
             data: node_info,
         };
-        loc_hashmap.insert(node_id as u32, meshnode);
+        meshnode_vec.push(meshnode);
     }
-    loc_hashmap
+    meshnode_vec
+}
+
+pub fn mock_edge_map_from_loc_info(
```

Comment:
This is a stub we'll call from the handlers. To mock, we can pretend that position packets are neighborinfo packets. Every time we get one, we'll use our existing location info (ids and snrs) to generate a graph with random edges.